src/git.spec.js

```diff
@@ -0,0 +1,11 @@
+import test from 'ava';
+import git from './git-commands';
+
+test('extract git versions return in expected format [, major, minor]', t => {
+  let [, major, minor] = git.versionArray('git version 2.10.1.windows.1');
+  t.is(major + '.' + minor, '2.10');
+  [, major, minor] = git.versionArray('git version 2.15.0');
+  t.is(major + '.' + minor, '2.15');
+  [, major, minor] = git.versionArray('git version 2.1.2 (Apple Git-101.1)\nhub version 2.2.9');

```

Comment:
I don't think the hub version will be possible to appear in the output, since aliases are only evaluated if we were to run bash in interactive mode via spawnsync.

Reply:
Just left this in to show we get the correct version number within a string of version numbers.